pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+line-length = 99
+target-version = ['py38']
```

Comment:
@mp24-git Is this file needed? or is this part of your personal local configuration? I believe we define some these line lengths and such for us in the `.flake8` file.

Reply:
I have removed it since pre-commit does the black formatting